operator/pkg/helm/helm.go

```diff
@@ -111,6 +113,11 @@ func renderChart(namespace, values string, chrt *chart.Chart, filterFunc Templat
 	}
 
 	caps := *chartutil.DefaultCapabilities
+
+	// overwrite helm default capabilities
+	operatorVersion, _ := chartutil.ParseKubeVersion("1." + strconv.Itoa(k8sversion.MinK8SVersion) + ".0")
```

Comment:
will this fail with some custom build like eks?

Reply:
no, k8sversion.MinK8SVersion is defined in https://github.com/istio/istio/blob/master/istioctl/pkg/install/k8sversion/version.go